lang/rust/rust-package.mk

```diff
@@ -15,26 +15,22 @@ ifeq ($(origin RUST_INCLUDE_DIR),undefined)
 endif
 include $(RUST_INCLUDE_DIR)/rust-values.mk
 
+CARGO_PKG_VARS= \
+	$(CARGO_PKG_CONFIG_VARS) \
+	CC=$(HOSTCC_NOCACHE) \
+	MAKEFLAGS="$(PKG_JOBS)"
+
 # $(1) path to the package (optional)
 # $(2) additional arguments to cargo (optional)
 define Build/Compile/Cargo
-	( \
-		cd $(PKG_BUILD_DIR) ; \
-		export PATH="$(CARGO_HOME)/bin:$(PATH)" ; \
-		CARGO_HOME=$(CARGO_HOME) \
-		TARGET_CFLAGS="$(TARGET_CFLAGS) $(RUSTC_CFLAGS)" \
-		TARGET_CC=$(TARGET_CC_NOCACHE) \
-		CC=$(HOSTCC_NOCACHE) \
-		RUSTFLAGS="$(CARGO_RUSTFLAGS)" \
-		$(CARGO_VARS) \
-		cargo install -v \
-			--profile stripped \
-			--target $(RUSTC_TARGET_ARCH) \
-			$(if $(strip $(RUST_PKG_FEATURES)),--features "$(strip $(RUST_PKG_FEATURES))") \
-			--root $(PKG_INSTALL_DIR) \
-			--path "$(if $(strip $(1)),$(strip $(1)),.)" \
-			$(2) ; \
-	)
+	+$(CARGO_PKG_VARS) \
+	cargo install -v \
+		--profile $(CARGO_PKG_PROFILE) \
+		$(if $(strip $(RUST_PKG_FEATURES)),--features "$(strip $(RUST_PKG_FEATURES))") \
+		--root $(PKG_INSTALL_DIR) \
+		--path "$(if $(strip $(1)),$(strip $(1)),$(PKG_BUILD_DIR))" \
```

Comment:
Keep the `cd $(PKG_BUILD_DIR)` so you can pass the relative paths to the `Build/Compile/Cargo` invocation.

Reply:
This is done, no need for `cd`.